src/cloud_provider/service.rs

```diff
@@ -944,7 +944,8 @@ where
             selector.as_str(),
             kubernetes.cloud_provider().credentials_environment_variables(),
         ),
-    )?;
+    )
+    .unwrap_or_else(|_| vec!["No Logs".to_string()]);
```

Comment:
Can you create a longer message plz, easier to find in the logs and more understandable (add context or explanation). Thx

Reply:
better ?